domains/auth/Auth.API/API/Models/DTOs/AcceptTermsDTO.cs

```diff
@@ -0,0 +1,6 @@
+namespace API.Models.DTOs;
+
+public record class AcceptTermsDTO
+{
+    public int Version { get; init; }
+}
```

Comment:
```suggestion
public record AcceptTermsDTO(int Version)
```

There will be more changes required tho